wot-wg-2023-draft.html

```diff
@@ -420,6 +418,10 @@ <h3>
             <!-- Since we plan to publish an update soon the URL might need to change -->
             Draft state: <a href="https://www.w3.org/TR/2020/NOTE-wot-scripting-api-20201124/">Group Note</a>
           </li>
+          <li>Binding Documents for Specific Protocol, Payload and Ecosystems (W3C Notes): These documents will take 
+            the requirements specified in <a href="binding-deliverable">Web of Things (WoT) Binding Templates</a> and 
```

Comment:
Bindings to other protocols and ecosystems should be motivated by use cases and requirements defined in the https://www.w3.org/TR/wot-usecases/.
At this point there are use cases for OPC-UA, MQTT, CoAP, LPWAN, Modbus, BACNet, Fieldbus. 
Are all these in scope?

Reply:
- Fieldbus is not a protocol as far as I know but a category.
- LPWAN is comparable to WiFi and too low level to have a binding for.
- The other protocols mentioned in the details section can be iterated on.

Reply:
I explicitly said that we should NOT include details in the charter, since the set of protocols we cover may change and be part of WG discussion.  The list of target protocols can be in the details (and it is).  Frankly we have enough "obvious" ones to get through that we won't even be able to get to all the industrially relevant ones.